app/test/frontend/handlers/_utils.dart

```diff
@@ -34,12 +34,22 @@ Future<shelf.Response> issueGetUri(Uri uri) async {
 }
 
 Future<String> expectHtmlResponse(shelf.Response response,
-    {int status = 200}) async {
+    {int status = 200, List<Pattern> present, List<Pattern> absent}) async {
   expect(response.statusCode, status);
   expect(response.headers['content-type'], 'text/html; charset="utf-8"');
   final content = await response.readAsString();
   expect(content, contains('<!DOCTYPE html>'));
   expect(content, contains('</html>'));
+  for (Pattern p in present ?? []) {
```

Comment:
Use default values in the parameter list instead...
`Future<String> expectHtmlResponse(shelf.Response response, {int status = 200, List<Pattern> present = [], List<Pattern> absent = []})`